aspnetcore/migration/Logging-nonaspnetcore.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Migrate from Microsoft.Extensions.Logging 2.1 to 2.2 or 3.0
+author: pakrym
+description: This article outlines the common steps for migrating a non-ASP.NET Core application that uses Microsoft.Extensions.Logging 2.1 to Microsoft.Extensions.Logging 2.2 or 3.0.
+ms.author: pakrym
+ms.custom: mvc
+ms.date: 01/04/2019
+uid: migration/logging-nonaspnetcore
+---
+
+# Migrate from Microsoft.Extensions.Logging 2.1 to 2.2 or 3.0
+
+This article outlines the common steps for migrating a non-ASP.NET Core application that uses `Microsoft.Extensions.Logging` 2.1 to `Microsoft.Extensions.Logging` 2.2 or 3.0.
```

Comment:
"app" favored over "application" ... I don't want to beat a :skull:🐴 tho. The last discussion was "where it makes sense;" but as I said during that discussion, that's pretty much 100% of the cases when "application" isn't part of a proper noun.